lib/pymedphys/_dicom/coords.py

```diff
@@ -150,30 +150,30 @@ def xyz_axes_from_dataset(
     if coord_system.upper() in ("FIXED", "IEC FIXED", "F"):
         x = x_dicom_fixed
         y = z_dicom_fixed
-        z = -np.flip(y_dicom_fixed)
```

Comment:
So I'm embarrassed to say that I'm pretty sure all of these flips were unnecessary in `xyz_axes_from_dataset`. I figured it out whilst exploring plotting and noted some inconvenient relationships between fixed and patient-based coord systems.

I'll try to think up a compelling way to demonstrate that it's working completely correctly now (more for my own peace of mind than anything!). Viewing my new test should hopefully help.

The idea here is that we want, say, `y_axis[0]` to always correspond to `dose[0, i, j]` (IEC) or `dose[k, 0, j]` (DICOM). With the flips, we'll instead get `y_axis[0]` sometimes corresponding with `dose[-1, i, j]` (IEC) depending on the orientation.

- Another way to think about this is that, without flips, the indices of the axes that correspond to the origin (or closest to), x[i0], y[j0] and z[k0] will be associated with dose[`k0`, `i0`, `j0`] (DICOM). This is regardless of orientation and coordinate system. You would still get an axis _swap_ between DICOM and IEC or between S/P and DL/DR, but the index itself is still that of the origin voxel.

Reply:
Incidentally, the flips were what necessitated some of the less unintuitive code in `xyz_axes_from_dataset()`. Removing flips provides an opportunity to rewrite to be clearer.